### DISCUSSION TITLE
Artwork for Radio stations

### DISCUSSION THREAD

**Discussion by jeroen-github** [music-assistant/support#1871] [Q&A] [answered]
Hi All,

I have tried to find info, if its possible and how to Add artwork to my Radio Station streams, unfortunately I was not able to find any info on this.
Would be great if anyone could help me on this.

**Comment by OzGav** [comment, marked as answer]
https://discord.com/channels/753947050995089438/1098939316861935698/1199057477686284399

**Reply by jeroen-github** [reply]
Unfortunately I'n new to discord, I created an account, but when I click on the above. I get an error
"Messages Failed to Load
![Screenshot 2024-02-06 200358](https://github.com/music-assistant/hass-music-assistant/assets/31064573/006603c2-5a9b-483f-9f01-0e6d7ef5534f)
"
![Screenshot 2024-02-06 200358](https://github.com/music-assistant/hass-music-assistant/assets/31064573/05582bef-f040-4449-920c-4613eed64632)

**Reply by jeroen-github** [reply]
So now Im able to read the discord server topic, I Assume I should ask the question over there.

**Reply by OzGav** [reply]
Your question was answered in the link I provided so please don’t ask it again.
